python/dgl/backend/pytorch/tensor.py

```diff
@@ -290,6 +291,10 @@ def clamp(data, min_val, max_val):
 def replace_inf_with_zero(x):
     return th.masked_fill(x, th.isinf(x), 0)
 
+def count_nonzero(input):
+    # TODO: fallback to numpy for backward compatibility
+    return np.count_nonzero(input)
```

Comment:
do we not need to convert it into numpy array first?

Reply:
It will convert implicitly.

Reply:
You don't have to.  numpy functions can work directly on PyTorch tensors.